### ISSUE TITLE
[Calendar- DatePicker v9]: need of Independent Calendar in fluent v9, just like fabric Calendar

### ISSUE THREAD

**Issue by nikitasingla-MSFT** [microsoft/fluentui#29374]
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

I want to use only the Calendar of the DatePicker component of fluent v9, instead of having a input box with a placeholder.

I just want the calendar of DatePicker along with goToToday button and other navigation 
<img width="442" alt="Screenshot 2023-10-03 at 6 19 04 PM" src="https://github.com/microsoft/fluentui/assets/118871240/fd4572ce-a62d-49c2-bce6-5124268d1297">

I want to use the Calendar in some popup box like below,  the following screenshot comprises of fabric Calendar
<img width="1728" alt="Screenshot 2023-10-03 at 3 35 51 PM" src="https://github.com/microsoft/fluentui/assets/118871240/47b8969e-3b86-4006-9663-abdd1f048c9b">

code snippet of fabric component and how we want to use it
<img width="668" alt="Screenshot 2023-10-03 at 6 46 21 PM" src="https://github.com/microsoft/fluentui/assets/118871240/a69bae93-5f31-46bf-b995-1d320d4bec93">

in short we want same functionality that fabric Calendar provides [https://fabricweb.z5.web.core.windows.net/oufr/8.110.11/#/examples/calendar](https://fabricweb.z5.web.core.windows.net/oufr/8.110.11/#/examples/calendar)

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Priority

High

**Comment by ling1726** [comment]
@sopranopillow as owner of datepicker-compat

**Comment by sopranopillow** [comment]
Tracking here #29438

**Comment by sopranopillow** [comment]
Fixed by #29540

**Comment by nikitasingla-MSFT** [comment]
@sopranopillow , I'm not able to see anything related in the documentation, is there any size prop??
